### PR TITLE
Release 1.0.1

### PR DESCRIPTION
**Pull request by ajlanghorn**
- Tightens dependency on vcloud-tools-tester
- Updates integration test details in README
- Update Gem ownership details

### CODE REVIEW

**Comment by dcarley** [comment]
Merged #161 so that we can roll it into the same release. Could you add an entry to the CHANGELOG?

**Comment by ajlanghorn** [comment]
@dcarley Done!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 1.0.1 (2015-01-29)
+
+  - Tighten dependency on vcloud-tools-tester
+  - Update integration test details in README
+  - Update Gem ownership details
+  - Refactored test namespacing
+
```
Comment (mattbostock) [inline CHANGELOG.md]: Could this be split into sections, e.g. 'Documentation', 'Bugfixes', etc?

Comment (ajlanghorn) [inline CHANGELOG.md]: Maybe. On payment of :cookie:.

**Comment by ajlanghorn** [comment]
@mattbostock How does that look?

**Comment by mattbostock** [comment]
Could you adjust the formatting of the section titles to match the rest of the changelog?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
+## 1.0.1 (2015-01-29)
+
+Documentation:
+
+    - Update integration test details in README
+    - Update Gem ownership details
+    - Update Gem ownership details
```
Comment (dcarley) [inline CHANGELOG.md]: Seeing double :eyeglasses:

Comment (ajlanghorn) [inline CHANGELOG.md]: Take off your glasses? ;)

**Comment by mattbostock** [comment]
:+1: :balloon:

**Comment by ajlanghorn** [comment]
:birthday:

**Pull request merged**